tests/triggers/test_external_task.py

```diff
@@ -438,3 +464,12 @@ def test_serialization(self):
             "execution_dates": [timezone.datetime(2022, 1, 1)],
             "poll_interval": 5,
         }
+
+
+def mocked_get_count(*args, **kwargs):
+    time.sleep(0.3)
+    return 1
+
+
+async def dummy_async_fun():
```

Comment:
We avoid to use `dummy` into the Apache Airflow repo (still exists in some places) so better use here `fake`, `mocked`, and etc.

Reply:
renamed to `fake`